omp/matrix/csr_kernels.cpp

```diff
@@ -126,6 +123,129 @@ GKO_INSTANTIATE_FOR_EACH_MIXED_VALUE_AND_INDEX_TYPE(
     GKO_DECLARE_CSR_ADVANCED_SPMV_KERNEL);
 
 
+/**
+ * Computes the begin offsets into A and B for the specific diagonal
+ */
+template <typename IndexType>
+inline void merge_path_search(
+    const int         diagonal,           ///< [in]The diagonal to search
+    const IndexType*        A,                  ///< [in]List A
+    const int         A_len,              ///< [in]Length of A
+    const int         B_len,              ///< [in]Length of B
```

Comment:
```suggestion
    const IndexType*        a,                  ///< [in]List A
    const int         a_len,              ///< [in]Length of A
    const int         b_len,              ///< [in]Length of B
```
we use snake case for variable (https://github.com/ginkgo-project/ginkgo/wiki/Contributing-guidelines#variables)